integ-test/src/test/java/org/opensearch/sql/legacy/JoinIT.java

```diff
@@ -288,6 +288,8 @@ public void hintMultiSearchCanRunFewTimesNL() throws IOException {
     Assert.assertThat(hits.length(), equalTo(42));
   }
 
+  // TODO: Fix joinWithGeoIntersectNL test when SQL_PAGINATION_API_SEARCH_AFTER is true
```

Comment:
Can we leave this TODO item for now?

Reply:
Any reason for this? can we capture it in an issue if not possible now?

Reply:
I have created this issue https://github.com/opensearch-project/sql/issues/2938 to fix this as follow-up.